cmd/ddev/cmd/addon-get.go

```diff
@@ -359,10 +359,11 @@ func getInjectedEnv(envFile string, verbose bool) string {
 		}
 		injectedEnv = "export"
 		for k, v := range envMap {
-			v = strings.Replace(v, " ", `\ `, -1)
+			// Escape all spaces and dollar signs
+			v = strings.ReplaceAll(strings.ReplaceAll(v, `$`, `\$`), ` `, `\ `)
```

Comment:
I'll leave it as it is here.

I don't really like how it's done, I copied the whole logic for `getInjectedEnv()` from another place (providers).

This should be either read from here:

```
set -a
source envFile
set +a
```

Or injected to `ddevapp.DockerEnv()`

But it's out of scope in this PR.

See my comment:

- https://github.com/ddev/ddev/issues/6701#issuecomment-2465023060